app/actions/handlers.py

```diff
@@ -145,9 +142,33 @@ async def action_pull_events(integration:Integration, action_config: PullEventsC
         )
         return [msg]
 
-        # Create a list of tasks.
-    tasklist = [asyncio.create_task(get_fire_alerts(integration, action_config, auth_config, aoi_data)),
-                 asyncio.create_task(get_integrated_alerts(integration, action_config, auth_config))
+    # Get AOI and Geostore data.
+    aoi_id = await dataapi.aoi_from_url(action_config.gfw_share_link_url)
+    aoi_data = await dataapi.get_aoi(aoi_id=aoi_id)
```

Comment:
This could be cleaned up, but the intent is to allow a user to provide a share link (even if it's a shortened URL). We follow it to it's underlying dashboard address to discover it's AOI ID.